src/era5grib_parallel/cdo_era5grib.py

```diff
@@ -48,85 +48,109 @@ def fields():
         "t, pressure, 130",
         "q, pressure, 133",
         "lsm, single, 172",
-        "z, single, 129"]
+        "z, single, 129",
+    ]
     return lines
 
-def repackage_grib(dt_string,outdir):
+
+def repackage_grib(dt_string, outdir):
     """
     For a single date/time, this function reads through the ERA5 netcdf archive
     to produce a GRIB file for UM driving model reconfiguration.
 
     Parameters
     ----------
     dt_string : string
-            The requested date to be repackaged in %Y-%m-%dT%H:%M:%S format 
+            The requested date to be repackaged in %Y-%m-%dT%H:%M:%S format
     outdir : Path
             The path for the output file to be written to
-            requested date to be repackaged in %Y-%m-%dT%H:%M:%S format 
+            requested date to be repackaged in %Y-%m-%dT%H:%M:%S format
 
     Returns
     -------
     None.  The GRIB file is left in the outdir directory.
     """
 
-    r = "%10.10f"%random()
+    r = "%10.10f" % random()
     r = r.replace("0.", "")
-    
+
     dt = datetime.strptime(dt_string, "%Y-%m-%dT%H:%M:%S")
     compact_dt_string = datetime.strftime(dt, "%Y%m%d%H%M.t+000")
-    
+
     os.chdir(outdir)
-    
+
     merge_files = []
 
     lines = fields()
 
     for line in lines:
-        var, stype, eccode = line.replace(' ', '').split(',')
+        var, stype, eccode = line.replace(" ", "").split(",")
         eccode = int(eccode)
-    
-        filedir = ERADIR + "/%s-levels/reanalysis/%s/%4.4d/"%(stype, var, dt.year)
+
+        filedir = ERADIR + "/%s-levels/reanalysis/%s/%4.4d/" % (stype, var, dt.year)
 
         files = os.listdir(filedir)
-        yyyymm = "%4.4d%2.2d"%(dt.year, dt.month)
+        yyyymm = "%4.4d%2.2d" % (dt.year, dt.month)
         fname = [f for f in files if yyyymm in f][0]
 
         # Select out the specific date/time from the netcdf archive file
         outfname = var + "_" + compact_dt_string + "_" + r + ".nc"
-        cmd = "cdo --eccodes seldate," + dt_string + " " + filedir + "/" + fname + " " + outfname
+        cmd = (
+            "cdo --eccodes seldate,"
+            + dt_string
+            + " "
+            + filedir
+            + "/"
+            + fname
+            + " "
+            + outfname
+        )
```

Comment:
Maybe a formatted string would work better here - I understand that `ruff` is doing their best with the formatting, but maybe something like:
```suggestion
        cmd = f"cdo --eccodes seldate,{dt_string} {filedir}/{fname} {outfname}
```
instead?

Reply:
This PR was not tackling formatting issues in the main scripts, but your comment make sense. 
I am going to change this here.